openfisca_uk/variables/demographic/person.py

```diff
@@ -130,7 +106,7 @@ class is_eldest_child(Variable):
     definition_period = YEAR
     value_type = bool
 
-    def formula(person, period, parameters):
+    def formula(person, period):
```

Comment:
Are we establishing this guideline, where parameters are skipped as an arg if unused?

Reply:
I think no - reverted, but will leave outside this PR to check for any others.